qbittorrent.json

```
@@ -0,0 +1,64 @@
+{
+    "qBittorrent": {
+        "containers": {
+            "qbittorrent-linuxserver.io": {
+                "image": "linuxserver/qbittorrent",
+                "launch_order": 1,
+                "opts": [
+                    [
+                        "-e",
+                        "UMASK_SET=007"
+                    ],
+                    [
+                        "-e",
+                        "WEBUI_PORT=8090"
+                    ]
+                ],
+                "ports": {
+                    "8090": {
+                        "description": "qBittorrent WebUI port. When changing this you must also change environment \"WEBUI_PORT\" setting.",
+                        "host_default": 8090,
```

Must match container port set above.
```suggestion
                        "host_default": 12500,
```